GetARock/GetARock/Screen/AddGathering/AddGatheringViewController.swift

```diff
@@ -42,7 +46,38 @@ class AddGatheringViewController: UIViewController {
     // MARK: - Method
 
     @IBAction func cancelButtonAction(_ sender: UIBarButtonItem) {
-        dismiss(animated: true)
+        if introductionTextView.text.count > 0 || (titleTextField?.text ?? "").count > 0 || gatheringLocation != nil {
+            let cancelAlert = UIAlertController(title: nil, message: "작성중인 내용이 있습니다. 작성을 취소하시겠습니까?", preferredStyle: .alert)
```

Comment:
p3;

120자가 넘어가서 줄바꿈이 필요해보여요!

```suggestion
            let cancelAlert = UIAlertController(
                title: nil, 
                message: "작성중인 내용이 있습니다. 작성을 취소하시겠습니까?", 
                preferredStyle: .alert
            )
```